tests/pmDownloadTest.php

```diff
@@ -7,6 +7,9 @@
   */
 class pmDownloadCase extends CommandUnishTestCase {
   public function testPmDownload() {
+    if (UNISH_DRUPAL_MAJOR_VERSION == '6') {
+      $this->markTestSkipped("pm-* no longer supported with Drupal 6.");
```

Comment:
What motivated this? I dont have a problem with it per se but its a little odd to keep working on D6 support and neutering it at same time.

Reply:
There are no supported releases for Drupal 6 any longer, which makes the `pm-*` commands less useful. However, I think I went too far & may have introduced a problem that didn't formerly exist. Probably the only problem with `pm-*` is `pm-updatecode` and `pm-updatecodestatus`. I don't plan to fix these, but I shouldn't break things that are still working. I'll investigate further.

Reply:
Are there no SAs coming out for Drupal 6 contrib modules anymore? If so I guess it makes sense to stop support for those commands.

I'd like for pm-download to continue to work. Thats quite popular.

Reply:
I'm stepping back through the tests I marked as skipped and ensuring that they are skipped for a good reason. For example, the `generate make` test relies on the bootstrap theme, which isn't available for some Drupal versions. This could be fixed, but I'm just skipping it because I don't want to maintain pm or make.

I'm going to avoid breaking them, though.

Reply:
> Are there no SAs coming out for Drupal 6 contrib modules anymore?

The D6LTS vendors publish security releases, but they are not in the update status data from Drupal.org, and there isn't a "generic" source of update status information.

myDropWizard (the company I work for) publishes alternative update status information which includes the D6LTS releases we create and it can be used by Drush via [a module](https://www.drupal.org/project/mydropwizard). But I'm not sure if that can really be used in the Drush tests, or if it's appropriate to rely on vendor-specific update status info...

Reply:
We could rely on that module in the tests, and include a pointer to it in the documentation. Let's leave that for a follow-on issue that we can do if desired.

It turned out to be unnecessary to skip most of these tests. I clarified why I was skipping the ones that remain skipped.